{"capabilityDescription":{"en":"Deletes a file from the filesystem.","zh":"Deletes a file from the filesystem."},"properties":{"Filename":{"en":{"displayName":"Filename","description":"The name of the file to delete."},"zh":{"description":"The name of the file to delete.","displayName":"文件名"}},"Directory Path":{"en":{"displayName":"Directory Path","description":"The path to the directory the file to delete is located in."},"zh":{"description":"The path to the directory the file to delete is located in.","displayName":"Directory Path"}}},"relationships":{"success":{"en":"All FlowFiles, for which an existing file has been deleted, are routed to this relationship","zh":"All FlowFiles, for which an existing file has been deleted, are routed to this relationship"},"failure":{"en":"All FlowFiles, for which an existing file could not be deleted, are routed to this relationship","zh":"All FlowFiles, for which an existing file could not be deleted, are routed to this relationship"},"not found":{"en":"All FlowFiles, for which the file to delete did not exist, are routed to this relationship","zh":"All FlowFiles, for which the file to delete did not exist, are routed to this relationship"}},"tags":{"en":["file","remove","delete","local","files","filesystem"],"zh":["文件","去除","删去","地方的","文件夹","文件系统"]},"writeAttributes":{"DeleteFile.failure.reason":{"en":"Human-readable reason of failure. Only available if FlowFile is routed to relationship 'failure'.","zh":"Human-readable reason of failure. Only available if FlowFile is routed to relationship 'failure'."},"DeleteFile.failure.exception.message":{"en":"The message of the exception thrown during processor execution. Only available if an exception caused the FlowFile to be routed to relationship 'failure'.","zh":"The message of the exception thrown during processor execution. Only available if an exception caused the FlowFile to be routed to relationship 'failure'."},"DeleteFile.failure.exception.class":{"en":"The class name of the exception thrown during processor execution. Only available if an exception caused the FlowFile to be routed to relationship 'failure'.","zh":"The class name of the exception thrown during processor execution. Only available if an exception caused the FlowFile to be routed to relationship 'failure'."}}}